packages/cells/src/model.ts

```diff
@@ -609,9 +609,11 @@ export class CodeCellModel extends CellModel implements ICodeCellModel {
       }
     }
     executionCount.changed.connect(this._onExecutionCountChanged, this);
+    executionCount.changed.connect(this.onModelDBExecutionsCountChange, this);
```

Comment:
Why do you register another handler for `executionCount.changed`?

Reply:
If you do this, you also need to unregister the event when dispose is called.

Reply:
Why not there, but in metadata, it's fine?
https://github.com/jupyter-rtc/jupyterlab/blob/73aeb1f9cac25eebf14237bb46346355483dcde3/packages/cells/src/model.ts#L180

Reply:
Got it. I'm on it!